lib/ansible/modules/packaging/language/pip.py

```diff
@@ -522,8 +583,8 @@ def main():
             cmd += ' %s' % extra_args
 
         if name:
-            for pkg in name:
-                cmd += ' %s' % _get_full_name(pkg, version)
+            for pkg in packages:
```

Comment:
Also, you could go for a generator comprehension with extending the command:
```python
cmd += ''.join(' "%s"' % dist for dist in distributions)
```

Reply:
P.S. In Python, we call installable bundles "distributions", we call `.py` files "modules" and we call folders, which contain `__init__.py` and other `.py` files "packages".
Let's not confuse the term.